src/pages/settings/Wallet/VerifyAccountPage.tsx

```diff
@@ -64,9 +65,9 @@ function VerifyAccountPage({route}: VerifyAccountPageProps) {
         if (navigateForwardTo) {
             Navigation.navigate(navigateForwardTo, CONST.NAVIGATION.TYPE.UP);
         } else {
-            Navigation.goBack();
+            Navigation.goBack(backTo);
```

Comment:
@FitseTLT Please add test step for this change

Reply:
Added Thx